lib/logger-helper.js

```diff
@@ -167,7 +167,7 @@ function shouldBeParsed(maskFields, excludeFields) {
 }
 
 function getResponseAudit(req, res, options) {
-    var headers = _.get(res, '_headers');
+    var headers = res && res._headers && 'function' === typeof res.getHeaders ? res.getHeaders() : undefined;
```

Comment:
As `res._headers` usage is deprecated starting from node 12, it's replaced with `res.getHeaders()` call.

Reply:
returning `undefined` is exactly the same behavior with how `_.get(res, '_headers')` works.